src/libs/fileDownload/types.ts

```diff
@@ -0,0 +1,9 @@
+import {Asset} from 'react-native-image-picker';
+
+type FileDownload = (url: string, fileName: string) => Promise<void>;
+type GetImageResolution = (url: File | Asset) => Promise<{width: number; height: number}>;
+
+type SplitExtensionFromFileName = (fileName: string) => {fileName: string; fileExtension: string};
```

Comment:
Could you extract `{fileName: string; fileExtension: string}` into a separate type?